modules/authorization-xacml/src/main/java/org/opencastproject/authorization/xacml/XACMLAuthorizationService.java

```diff
@@ -393,19 +392,15 @@ private File fromWorkspace(URI uri) {
   /** Load an ACL from the given URI. */
   private Option<AccessControlList> loadAcl(final URI uri) {
     logger.debug("Load Acl from {}", uri);
-    final File file = fromWorkspace(uri);
-    if (file != null) {
-      try {
-        InputStream in = new FileInputStream(file);
-        AccessControlList acl = XACMLUtils.parseXacml(in);
-        if (acl != null) {
-          return Option.option(acl);
-        }
-      } catch (Exception e) {
-        logger.error("Exception occured: {}", e);
+    try (InputStream is = workspace.read(uri)) {
+      AccessControlList acl = XACMLUtils.parseXacml(is);
+      if (acl != null) {
+        return Option.option(acl);
       }
-    } else {
+    } catch (NotFoundException e) {
       logger.debug("URI {} not found", uri);
+    } catch (Exception e) {
+      logger.error("Exception occured: {}", e);
```

Comment:
Since I recently fixed this in one of my PRs: The {} are unnecessary when logging just the exception, this would result in "Exception occured: {}" followed by the stack trace instead of the desired "Exception occured: "

Reply:
WARN log level has been used in the past for very bad errors that don't crash critical services. ERROR is more of an all-hands-on-deck log level. I realize the log level was already at ERROR before this pull, but since it's being touched now, it's something to consider.